classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label
```

Comment:
- [ ] please make sure to set up the correct attribute reader for Student as well in order for the relationship to work.

Reply:
Thanks for the feedback, @mikethreels, but I don't get what you mean here. 

I have set the attribute accessor for Student to `:classroom`. Could you please, explain further?